style/styles.css

```diff
@@ -597,4 +634,170 @@ input:checked + .slider:before {
     .bizzNESTLogoStyle img {
         max-width: 50%; /* Even smaller for phones */
     }
+    .modalComponent {
+        width: 95%; /* Full width on very small screens */
+        padding: 15px; /* Less padding for mobile */
+    }
+
+    .topTextHeader {
+        font-size: 24px; /* Smaller font size for mobile */
+    }
+
+    .subtext {
+        font-size: 12px; /* Adjust subtext for mobile */
+    }
+    .dropdown {
+        width: 100%; /* Ensure dropdowns are full width */
+        margin-top: 0; /* Add margin for separation */
+    }
+
+    .dropdown button {
+        width: 100%;
+    }
+    .searchbox-container {
+        width: 95%; /* Full width on very small screens */
+        margin: 0;
+    }
+
+    .checkboxDiv {
```

Comment:
I would recommend not adding comments to every rule. You should put a comment for a rule when necessary.